numpy/random/philox.pyx

```diff
@@ -163,138 +158,41 @@ cdef class Philox:
-            Output shape.  If the given shape is, e.g., ``(m, n, k)``, then
-            ``m * n * k`` samples are drawn.  Default is None, in which case a
-            single value is returned.
-        output : bool, optional
-            Output values.  Used for performance testing since the generated
-            values are not returned.
-
-        Returns
-        -------
-        out : uint or ndarray
-            Drawn samples.
-
-        Notes
-        -----
-        This method directly exposes the the raw underlying pseudo-random
-        number generator. All values are returned as unsigned 64-bit
-        values irrespective of the number of bits produced by the PRNG.
-
-        See the class docstring for the number of bits returned.
-        """
-        return random_raw(&self._bitgen, self.lock, size, output)
-
-    def _benchmark(self, Py_ssize_t cnt, method=u'uint64'):
-        return benchmark(&self._bitgen, self.lock, cnt, method)
-
-    def seed(self, seed=None, counter=None, key=None):
-        """
-        seed(seed=None, counter=None, key=None)
-
-        Seed the generator.
-
-        This method is called when ``Philox`` is initialized. It can be
-        called again to re-seed the generator. For details, see
-        ``Philox``.
-
-        Parameters
-        ----------
-        seed : int, optional
-            Seed for ``Philox``.
-        counter : {int array}, optional
-            Positive integer less than 2**256 containing the counter position
-            or a 4 element array of uint64 containing the counter
-        key : {int, array}, options
-            Positive integer less than 2**128 containing the key
-            or a 2 element array of uint64 containing the key
-
-        Raises
-        ------
-        ValueError
-            If values are out of range for the PRNG.
-
-        Notes
-        -----
-        The two representation of the counter and key are related through
-        array[i] = (value // 2**(64*i)) % 2**64.
-        """
-        if seed is not None and key is not None:
-            raise ValueError('seed and key cannot be both used')
-        ub = 2 ** 64
-        if key is None:
-            if seed is None:
-                try:
-                    state = random_entropy(4)
-                except RuntimeError:
-                    state = random_entropy(4, 'fallback')
-                state = state.view(np.uint64)
-            else:
-                state = seed_by_array(seed, 2)
-            for i in range(2):
-                self.rng_state.key.v[i] = state[i]
-        else:
-            key = int_to_array(key, 'key', 128, 64)
-            for i in range(2):
-                self.rng_state.key.v[i] = key[i]
-        counter = 0 if counter is None else counter
-        counter = int_to_array(counter, 'counter', 256, 64)
-        for i in range(4):
-            self.rng_state.ctr.v[i] = counter[i]
-
-        self._reset_state_variables()
-
```

Comment:
The `ctypes` and `cffi` properties can also be deleted, as they are provided by `BitGenerator`.

Reply:
removing